src/server/event-stream-manager.ts

```diff
@@ -126,6 +126,8 @@ class EventStreamManager {
       JSON.stringify(obj),
     )
 
+    log.info('EventStreamManager', `emitting ${EventTypeRev[type]} on ${new Date().toString()}`)
```

Comment:
Maybe we should skip `message` event.

Reply:
if we keep it we can better track the problem when miaohui misses a message.